content/F2015/F15-event-orientation.md

```diff
@@ -0,0 +1,25 @@
+title: Orientation Event and Dinner
+Date: 2015-09-13 15:00-18:00
+Category: Events
+Tags: social, big csters, academic
+Slug: orientation-F15
+Author: Fatema Boxwala
+Summary: Come out and learn about the university life, and get to know some
+awesome women in CS!
+
+Welcome to Waterloo, incoming women in CS! To help you get used to life here
```

Comment:
I am wondering if we'd like to explicitly define women here like we do in Big CSters? Or at the very least provide a link to the FAQ!